app/views/projects/_search_form.html.erb

```diff
@@ -4,10 +4,18 @@
     <% [:platforms, :licenses, :languages, :keywords, :sort].each do |field| %>
       <%= hidden_field_tag field, params[field] if params[field].present? %>
     <% end %>
+
     <span class="input-group-btn">
       <button class="btn btn-primary" type="submit">
         <%= fa_icon 'search' %>
       </button>
     </span>
+
+    <% if logged_in? && (current_user.admin? || current_user.current_api_key.is_internal?) %>
```

Comment:
Maybe add an `admin_or_internal?` method on `User` that does the two checks that could be used here?